apis/apps/v1alpha1/cloneset_types.go

```diff
@@ -93,6 +93,9 @@ type CloneSetScaleStrategy struct {
 	// The scale will fail if the number of unavailable pods were greater than this MaxUnavailable at scaling up.
 	// MaxUnavailable works only when scaling up.
 	MaxUnavailable *intstr.IntOrString `json:"maxUnavailable,omitempty"`
+
+	// Enable cloneset to clean up the useless pvc when pod hangs in terminating state.
+	CleanUpPVC bool `json:"cleanUpPVC,omitempty"`
```

Comment:
How about `DisablePVCReuse`? So that controller will not reuse existing PVCs when creating new Pods and cleanup those useless PVCs whose Pods has already been deleted or in Succeeded/Failed phases.

Reply:
@zmberg @FillZpp Fi I update the pr with following the comments.